### PR TITLE
Add ecosystem, raft, performance and transaction SIGs

### PR DESCRIPTION
**Pull request by zhangjinpeng1987**
Signed-off-by: zhangjinpeng1987 <zhangjinpeng@pingcap.com>

Add ecosystem, raft, performance and transaction SIGs.

- [x] tech leader
- [x] english workflow

### CODE REVIEW

File: sig/raft/constitution-zh_CN.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Raft SIG 章程
+
+本章程遵循 TiKV Community 章程和制度，使用 [SIG Governance](/GOVERNANCE-zh_CN.md) 中的角色定义以及职责。
+
+## 职责范围
+
+Raft SIG 的主要职责是致力于分布式一致性算法 Raft 库的发展，以及 Raft 在 TiKV 中的应用和优化，并组织社区成员进行相关开发和维护。工作范围包括为 TiKV 引入新的存储引擎，以及对现有存储引擎进行稳定性、性能、易用性等方面的提升。仅 Active Contributor 或更高等级社区成员可成为 SIG 成员并参与到相应事项中来，但 SIG 各项实行的决策和进行的项目将公开。
```
Comment (NingLin-P) [inline sig/raft/constitution-zh_CN.md]: `工作范围包括为 TiKV 引入新的存储引擎，以及对现有存储引擎进行稳定性、性能、易用性等方面的提升` seem not suitable to appear here.

Comment (zhangjinpeng1987) [reply sig/raft/constitution-zh_CN.md]: Thx

File: sig/transaction/membership.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Transaction SIG Members
+
+## Tech Leader
```
Comment (youjiali1995) [inline sig/transaction/membership.md]: [@MyonKeminta](https://github.com/MyonKeminta)
[@nrc](https://github.com/nrc)

Comment (zhangjinpeng1987) [reply sig/transaction/membership.md]: done

File: sig/transaction/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+- Constitution: [中文](./constitution-zh_CN.md)
+- Membership: [English](./membership.md)
+- Workflow: [中文](./workflow-zh_CN.md)
+- Document Archive: 中文/English
+- Meeting Notes: English
+
+## Meetings
+
+* Regular SIG Meeting: 
+     * Bi-weekly on Wed 10am, Beijing Time (Tue 6pm PST) starting from 2020/02/12.
```
Comment (MyonKeminta) [inline sig/transaction/README.md]: Is this correct?

Comment (zhangjinpeng1987) [reply sig/transaction/README.md]: Will change it after you guys determined the meeting time.

File: sig/raft/membership.md [hunk 4]
```diff
@@ -0,0 +1,23 @@
+# Raft SIG Members
+
+## Tech Leader
+
+- [@BusyJay](https://github.com/BusyJay)
```
Comment (BusyJay) [inline sig/raft/membership.md]: Name @hicqu as tech leader too.

Comment (zhangjinpeng1987) [reply sig/raft/membership.md]: I will add it after this big pr merged.

Comment (zhangjinpeng1987) [reply sig/raft/membership.md]: Done

File: sig/raft/constitution-zh_CN.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
+# Raft SIG 章程
+
+本章程遵循 TiKV Community 章程和制度，使用 [SIG Governance](/GOVERNANCE-zh_CN.md) 中的角色定义以及职责。
+
+## 职责范围
+
+Raft SIG 的主要职责是致力于分布式一致性算法 [Raft](https://github.com/tikv/raft-rs) 库的发展，以及 Multi-Raft 在 TiKV 中的应用和优化，并组织社区成员进行相关开发和维护。仅 Active Contributor 或更高等级社区成员可成为 SIG 成员并参与到相应事项中来，但 SIG 各项实行的决策和进行的项目将公开。
```
Comment (gengliqi) [inline sig/raft/constitution-zh_CN.md]: In my opinion, `consensus` should be translated to `共识`. 
`一致性` means consistency.

Comment (zhangjinpeng1987) [reply sig/raft/constitution-zh_CN.md]: ok

Comment (zhangjinpeng1987) [reply sig/raft/constitution-zh_CN.md]: done

**Pull request merged**